vmdk_plugin/Makefile

```diff
@@ -287,7 +287,7 @@ CLEANESX_SH   := $(DEPLOY_TOOLS) cleanesx
 deploy-esx: clean-esx
 	$(DEPLOY_ESX_SH) "$(ESX)" "$(VIB_BIN)"
 
-VMS= $(VM1) $(VM2) $(WORKER2)
+VMS= $(VM1) $(VM2) $(WORKER2) $(CI_NODE4)
```

Comment:
Yeah all are VM env vars. `WORKER2 ` meant for the third VM node part of swarm cluster. `CI_NODE4 ` was added to simplify installation process. 

this will be simplified further as part of https://github.com/vmware/docker-volume-vsphere/issues/1274.